guides/customization/tailscale.md

```diff
@@ -78,48 +84,37 @@ the target files, in a subdirectory called `files`:
 ```
 
 While it is possible to configure everything directly in a single Dockerfile, we
-recommend using a folder hierarchy, since it makes it easier to create a
+recommend using a folder hierarchy, since this makes it easier to create a
 reproducible image and examine the change history for individual files.
 
-Create the folder hierarchy using:
+Create the folder hierarchy:
 
 ```console
-$ mkdir --parents --verbose \
+mkdir --parents --verbose \
```

Comment:
if we remove the output, let's remove --verbose from the example too (console and shell-session are meant to show the terminal prompt, input command, and command results, so that users can confirm that the command ran as we expect)

I can't find a great reference right now, but did find this https://stackoverflow.com/a/49004070